tests/test_service.py

```diff
@@ -58,6 +58,13 @@ def test_hello(self):
         self.assertEqual(resp.status_code, HTTP_200_OK)
 
 
+    def test_get_supplier_list(self):
+        """ Get a list of Suppliers """
+        resp = self.app.get('/suppliers')
+        self.assertEqual(resp.status_code, HTTP_200_OK)
+        self.assertTrue(len(resp.data) > 0)
```

Comment:
Why we can get supplier from DB without creating objects. DB will be reset before running each test case. Here we should call ```_create_suppliers(count)``` to create supplier object in batch, then get list and check whether the amount of suppliers is equal to ```count```. In ```bdd``` repo we have ```data_load``` in setup, that's why we have several suppliers initially without creating. We have deleted ```data_load``` part and we should follow the method in ```tdd``` repo. ie. first create objects using factory method.